pydantic/fields.py

```diff
@@ -679,20 +681,20 @@ def Field(  # noqa: C901
         multiple_of: Value must be a multiple of this. Only applicable to numbers.
         min_length: Minimum length for strings.
         max_length: Maximum length for strings.
-        pattern: Pattern for strings.
+        pattern: Pattern for strings (a regular expression).
```

Comment:
Should we say anything here about configuring the `regex_engine`? Fine if not, bc I believe that's at the `model_config` level.

Reply:
sounds good to me